h/static/scripts/group-forms/utils/test/annotation-metadata-test.js

```diff
@@ -0,0 +1,101 @@
+import { pageLabel, quote, username } from '../annotation-metadata';
```

Comment:
Do you plan to move any of these into the `@hypothesis/annotation-ui` package or another package?